management-account/terraform/sso-admin-permission-sets.tf

```diff
@@ -424,6 +424,25 @@ data "aws_iam_policy_document" "modernisation_platform_engineer" {
     ]
     resources = ["arn:aws:dynamodb:eu-west-2:${coalesce(local.modernisation_platform_accounts.modernisation_platform_id...)}:table/modernisation-platform-terraform-state-lock"]
   }
+  statement {
+    effect = "Allow"
```

Comment:
Could we add in a `sid` such as `sid = "ManageModernisationPlatformAccounts"` or something along those lines 😁